packages/databases-collections-list/src/items-grid.tsx

```diff
@@ -88,7 +88,7 @@ type ItemsGridProps<T> = {
   renderItem: RenderItem<T>;
 };
 
-const CONTROLS_HEIGHT = spacing[5] + 36;
+const CONTROLS_HEIGHT = (spacing[5] as number) + 36;
```

Comment:
Isn't the `spacing` type already a number?

Reply:
eslint doesn't allow me to use `+` here without it anymore 🤷

Reply:
Huh weird. I feel like there are other places in the code where we do similar. - could it be lint in your editor maybe not picking something up? Sometimes my eslint acts up and I have to restart vscode to fix it.